### DISCUSSION TITLE
AssetMapper

### DISCUSSION THREAD

**Discussion by soufiiyane** [symfony/symfony#54284] [Q&A] [answered]
Hello guys can i use AssetMapper with TypeScript ? also do you recommend assetMapper over WebPack ?

**Comment by lyrixx** [comment, marked as answer]
If you need TS then you must compile your files. So webpack is strongly recommend

**Reply by soufiiyane** [reply]
Thank you !  really had so much trouble setting it up with assetMapper, i will switch to WebPack instead

**Reply by stof** [reply]
If you want to stick to AssetMapper, https://packagist.org/packages/sensiolabs/typescript-bundle might interest you.
